contracts/PolygonTokenBridger.sol

```diff
@@ -90,6 +90,8 @@ contract PolygonTokenBridger is Lockable {
         uint256 _l1ChainId,
         uint256 _l2ChainId
     ) {
+        require(_destination != address(0), "invalid dest address");
+        require(_l2WrappedMatic != address(0), "invalid wmatic address");
```

Comment:
I'm not sure about the value of these checks. 0 is only one possible invalid address, but there are ~infinitely many other invalid addresses as well. The only way we can really be confident that the address is set correctly is via post-deployment verification.

I'm not firmly against it or anything, it just seems a bit redundant and might give a false sense of security.